src/rgw/rgw_sal_motr.cc

```diff
@@ -1899,7 +1882,14 @@ int MotrObject::MotrReadOp::iterate(const DoutPrefixProvider* dpp, int64_t off,
 
 int MotrObject::MotrReadOp::get_attr(const DoutPrefixProvider* dpp, const char* name, bufferlist& dest, optional_yield y)
 {
-  //return 0;
+  if (source == nullptr)
```

Comment:
This code can be as below:
  if (source != nullptr) {
     rgw::sal::Attrs &attrs = source->get_attrs();
     auto iter = attrs.find(name);
     if (iter != attrs.end()) {
         dest = iter->second;
         return 0;
      }
  }
  return -ENODATA;

Reply:
Hi Sachin,
this will be addressed in the future PR.